FAQ.md

```diff
@@ -16,6 +18,20 @@ Summarize the question and quote the reply, linking back to the original comment
 * [Can I put the manifest and lock in the vendor directory?](#can-i-put-the-manifest-and-lock-in-the-vendor-directory)
 * [Why did dep use a different revision for package X instead of the revision in the lock file?](#why-did-dep-use-a-different-revision-for-package-x-instead-of-the-revision-in-the-lock-file)
 
+## What is the difference between the manifest and lock?
```

Comment:
Let's revise to

> "What is the difference between Gopkg.toml (the "manifest") and Gopkg.lock (the "lock")?"

That way we don't assume people are familiar with the abstract terminology of manifest/lock